packages/components/src/internal/constants.ts

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 import { Set } from 'immutable';
+import { getServerContext } from '@labkey/api';
+
+export const LABKEY_VIS = getServerContext().vis;
```

Comment:
As we discussed, I would recommend adding a note here about why these constants are declared.

Reply:
done